game/core/package.json

```diff
@@ -72,7 +72,8 @@
         "uuid": "^10.0.0"
     },
     "devDependencies": {
-        "@parcel/config-default": "^2.12.0",
+        "@babel/preset-typescript": "^7.24.7",
+        "globals": "^15.9.0",
```

Comment:
What is it needed for? Also yarn.lock has to be updated, I do not see it has been committed

Reply:
messed up with the changes for the file, thanks for pointing!